packages/angular-workspace/nimble-angular/table/testing/tests/table-pageobject.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
```

Comment:
Could we name this file `table.pageobject.spec.ts`. I think that better aligns with our conventions for `file.ts` and `tests/file.spec.ts`